extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -209,18 +233,23 @@ protected function getNolangTxt() {
 	 * By the way, if every document on the site was as big as the Jim Henson page,
 	 * then it would take under two minutes to parse them all using this function.
 	 * So this scales on the application side. I promise. I mathed it.
+	 *
 	 * @param string $html
+	 *
 	 * @return array
 	 */
 
-	
 	protected function prepValuesFromHtml( $html ) {
-		$result = array();
-		$paragraphs = array();
-		
-		$html = str_replace(["&lt;", "&gt;"], "", $html); // workaround for bug in html_entity_decode that truncates the text
+		$result = [];
+		$paragraphs = [];
+
+		$html = str_replace(
+			[ "&lt;", "&gt;" ],
+			"",
+			$html
+		); // workaround for bug in html_entity_decode that truncates the text
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)